llvm/test/CodeGen/AArch64/arm64-narrow-st-merge.ll

```diff
@@ -7,6 +8,19 @@
 ; CHECK-STRICT: strh wzr
 ; CHECK-STRICT: strh wzr
```

Comment:
The script won't touch checks outside actual methods - you will need to remove these manually.